_includes/latest-posts.html

```diff
@@ -240,20 +256,26 @@ <h3 class="latest-posts__list-item-title">
             <span class="latest-posts__list-item-summary-readmore">➟ <u>{% if post.lang == 'da_dk' %}Læs&nbsp;mere{% else %}Read&nbsp;more{% endif %}</u></span>
           </div>
           <div class="latest-posts__meta">
-            <img src="{{post.author.image}}" alt="" class="latest-posts__meta-avatar" />
-            <div class="latest-posts__meta-author">
-              {% if post.author.name %}
-                {% if post.lang == 'da_dk' %}Af{% else %}By{% endif %} {{ post.author.name }}
-              {% endif %}
-              {% if post.author.title %}
-                <i>{{ post.author.title }}</i>
-              {% endif %}
-              {% if post.at %}
-                <span class="latest-posts__meta-date">
-                  Published {{ post.at }}
-                </span>
-              {% endif %}
+            <h4>{% if post.lang == 'da_dk' and post.author.size == 1 %}Forfatter{% elsif post.author.size == 1 %}Author{% elsif post.lang == 'da_dk' and post.author.size > 1 %}Forfattere{% elsif post.author.size > 1 %}Authors{% endif %}</h4>
+            <div class="latest-posts__meta-authors-wrapper">
+              {% assign authors = post.author %}
+              {% for author in authors %}
+              <div class="latest-posts__meta-author-wrapper">
+                <img src="{{author.image}}" alt="" class="latest-posts__meta-avatar" />
+                <div class="latest-posts__meta-author">
+                  {% if author.name %}
+                    {{ author.name }}
+                  {% endif %}
+                  {% if author.title %}
+                    <i>{{ author.title }}</i>
+                  {% endif %}
+                </div>
+              </div>
+              {% endfor %}
             </div>
+            <span class="latest-posts__meta-date">
+              {% if post.lang == 'da_dk' %}Offentliggjort{% else %}Published{% endif %} {{ post.at }}
+            </span>
```

Comment:
The class names here would benefit from being rethought.
I think the naming should go in the lines of:
- `latest-posts__meta` (same as before)
- `latest-posts__meta-header` (h4)
- `latest-posts__authors`
- `latest-posts__author` (the wrapper/container element for each author).
- `latest-posts__author-avatar`
- `latest-posts__author-text`
- `latest-posts__publish-date`

Simplifying and rethinking what each element represents in the UI.